tests/unit/source_code/test_notebook_linter.py

```diff
@@ -482,6 +482,14 @@ def test_notebook_linter_name():
                     end_line=6,
                     end_col=29,
                 ),
+                Advice(
+                    code='table-migrate',
+                    message='The default format changed in Databricks Runtime 8.0, from ' 'Parquet to Delta',
```

Comment:
```suggestion
                    message='The default format changed in Databricks Runtime 8.0, from Parquet to Delta',
```
remove extra space in the middle of the string - makes it confusing to read

Reply:
Fixed